### DISCUSSION TITLE
build on windows

### DISCUSSION THREAD

**Discussion by shehabso** [carbon-language/carbon-lang#4516] [Getting started]
Why did the Carbon maintainers decide not to create a .dev Docker container? 
i can do it .

**Comment by jonmeow** [comment]
The current contributors generally don't use docker containers, and so they're unmaintained and potentially broken (for example, #4065). For Windows in particular, note that the Windows build currently requires WSL per #298.

We'd be happy to review contributions with documentation on how to use it, but it will need people interested in using & maintaining it in order to stay working.
